client.go

```diff
@@ -1547,6 +1559,53 @@ func (c *ToznySDKV3) WriteFile(ctx context.Context, recordType string, plain map
 	return fileRecord, nil
 }
 
+type ReadFileOptions struct {
+	RecordID         string
+	DownloadFileName string
+}
+
+// ReadFile downloads and decrypts the file from the record
+func (c *ToznySDKV3) ReadFile(ctx context.Context, options ReadFileOptions) error {
+	secretUUID, err := uuid.Parse(options.RecordID)
```

Comment:
nit: this what types are for, if this function requires RecordID to be a UUID, it should type it as a uuid.UUID not a string